crates/sui-types/src/lib.rs

```diff
@@ -88,6 +93,12 @@ const fn get_hex_address_two() -> AccountAddress {
     AccountAddress::new(addr)
 }
 
+const fn get_hex_address_three() -> AccountAddress {
+    let mut addr = [0u8; AccountAddress::LENGTH];
+    addr[AccountAddress::LENGTH - 1] = 3u8;
+    AccountAddress::new(addr)
+}
```

Comment:
nit: Suggest implementing this in a slightly more generic way:

```suggestion
const fn address_from_single_byte(b: u8) -> AccountAddress {
    let mut addr = [0u8; AccountAddress::LENGTH];
    addr[AccountAddress::LENGTH - 1] = b;
    AccountAddress::new(addr)
}
```

This can be re-used for both of these cases, and probably to implement `ObjectID::from_single_byte` as well.